frontend/src/app/explore/page.tsx

```diff
@@ -20,7 +20,6 @@ import { BountyStatusBadgeGroup } from "../../components/bountyStatus";
 import { HasConnectedAccount } from "../../components/hasConnectedAccount";
 import { useBlockTimestamp } from "../../hooks/block";
 import { getBountyStatus } from "../../utils/bounty";
-import { GOOGLE_BOUNTY_CREATION_FORM_URL } from "../../utils/links";
```

Comment:
I think we can remove this bounty creation form URL from `src/app/utils/links`, no?

Reply:
Yeah! Good catch. Done!